### PR TITLE
J3576

### PR DESCRIPTION
**Pull request by rozza**
This has consistently passed over the last few runs:
https://evergreen.mongodb.com/version/5e18a6dd9ccd4e34f84e115a

### CODE REVIEW

File: .evergreen/.evg.yml [hunk 1]
```diff
@@ -157,6 +157,23 @@ functions:
         content_type: ${content_type|text/plain}
         display_name: "orchestration.log"
 
+  "upload working dir logs":
```
Comment (rozza) [inline .evergreen/.evg.yml]: This is to capture any hs_err_pid logs

File: .evergreen/.evg.yml [hunk 2]
```diff
@@ -394,6 +411,7 @@ post:
   # Removed, causing timeouts
   # - func: "upload working dir"
```
Comment (rozza) [inline .evergreen/.evg.yml]: Note: The full upload used to time out and was disabled some time ago.

File: build.gradle [hunk 3]
```diff
@@ -174,6 +174,7 @@ configure(javaCodeCheckedProjects) {
     /* Testing */
     tasks.withType(Test) {
         maxHeapSize = "3g"
+        jvmArgs = ['-XX:-UseCompressedOops']
```
Comment (jyemin) [inline build.gradle]: I'm surprised this helps.  It seems like this is the default as of Java 8:

    $ java -XX:+PrintFlagsFinal -version|grep UseCompressedOops
         bool UseCompressedOops                        := true                                {lp64_product}
         openjdk version "1.8.0_232"

File: .evergreen/.evg.yml (hunk 1 above)
Comment (jyemin) [reply .evergreen/.evg.yml]: Nice

File: build.gradle (hunk 3 above)
Comment (rozza) [reply build.gradle]: Yes this is to turn it off (its prefixed with a minus). Although the last rerun on evergreen just failed so it doesn't seem to help :(

**Comment by rozza** [comment]
I'm closing this as the last evergreen run threw a OOM!

**Pull request closed without merging**